### DISCUSSION TITLE
I can't compile tests

### DISCUSSION THREAD

**Discussion by 1113870631** [svpcom/wfb-ng#196] [Questions] [answered]
there are some error that I can't solve when I run command make 

![1](https://user-images.githubusercontent.com/61228813/153841144-c08dd152-cc92-4f10-bbb5-8dba00f3ae37.png)
![2](https://user-images.githubusercontent.com/61228813/153841324-50b71bff-31b8-4223-95f7-b5f44d5e94b2.png)
![image](https://user-images.githubusercontent.com/61228813/153841394-5d291c5d-059a-4d3b-946c-72488eea2236.png)

for the error: "No module named builtins"
I run command "pip install future" it not works 
would you help me ? thanks

**Comment by svpcom** [comment, marked as answer]
apt install python3-all

**Reply by 1113870631** [reply]
still don't work

**Reply by svpcom** [reply]
apt install python3-future python3-six
what wifibroadcast version are you building?
what is you linux distro?
Check that ./env/bin/python is python3

**Reply by 1113870631** [reply]
I use ubuntu 18.04
the branch is master
I have install the "Depends3: python3-twisted, libpcap-dev, libsodium-dev, python3-pyroute2, python3-future, python3-serial"
that list in the file stdeb.cfg
my python :/usr/bin/python -> /usr/bin/python3.6

I have a question about if my python were setted 3.6 correctly why the error information is python2.7 don't have module named builtins

is my python version don't set to 3.6 successfully?

**Reply by svpcom** [reply]
Fixed in last commit. Please try again

**Reply by 1113870631** [reply]
it's ok, thank you vary much
